### PR TITLE
Move action commands under installations 

### PR DESCRIPTION
**Pull request by carolynvs**
# What does this change
The bundle action commands (install, invoke, upgrade, uninstall) really act on installations, not bundles. I am moving those commands under installations, so that the full command for say "porter install" would be "porter installation install".

Everyone uses the aliases anyway so I don't expect this to be an impactful change to the commands people have automated. But where it does help is better group the commands in the help text. Now you can find upgrade associated to an installation, which is a better fit.

I've also fixed `mage DocsGen` so that it properly deletes doc files that are no longer needed.

# What issue does it fix
Part of #1430 

# Notes for the reviewer
N/A

# Checklist
- [ ] Did you write tests?
- [ ] Did you write documentation?
- [ ] Did you change porter.yaml or a storage document record? Update the corresponding schema file.
- [ ] If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️

# Reviewer Checklist
* Comment with /azp run test-porter-release if a magefile or build script was modified
* Comment with /azp run porter-integration if it's a non-trivial PR

[contributors]: https://getporter.org/src/CONTRIBUTORS.md

### CODE REVIEW

File: cmd/porter/installations.go [hunk 1]
```diff
@@ -206,3 +211,256 @@ func buildInstallationRunsListCommand(p *porter.Porter) *cobra.Command {
...
+
+Porter uses the docker driver as the default runtime for executing a bundle's invocation image, but an alternate driver may be supplied via '--driver/-d' or the PORTER_RUNTIME_DRIVER environment variable.
+For example, the 'debug' driver may be specified, which simply logs the info given to it and then exits.
+
+The docker driver runs the bundle container using the local Docker host. To use a remote Docker host, set the following environment variables:
+  DOCKER_HOST (required)
+  DOCKER_TLS_VERIFY (optional)
+  DOCKER_CERT_PATH (optional)
+`,
+		Example: `  porter bundle install
```
Comment (VinozzZ) [inline cmd/porter/installations.go:234]: The example commands need to be updated as well?

File: cmd/porter/installations.go [hunk 2]
```diff
@@ -206,3 +211,256 @@ func buildInstallationRunsListCommand(p *porter.Porter) *cobra.Command {
...
+Porter uses the docker driver as the default runtime for executing a bundle's invocation image, but an alternate driver may be supplied via '--driver/-d' or the PORTER_RUNTIME_DRIVER environment variable.
+For example, the 'debug' driver may be specified, which simply logs the info given to it and then exits.
+
+The docker driver runs the bundle container using the local Docker host. To use a remote Docker host, set the following environment variables:
+  DOCKER_HOST (required)
+  DOCKER_TLS_VERIFY (optional)
+  DOCKER_CERT_PATH (optional)
+`,
+		Example: `  porter bundle upgrade --version 0.2.0
+  porter bundle upgrade --reference ghcr.io/getporter/examples/kubernetes:v0.2.0
```
Comment (VinozzZ) [inline cmd/porter/installations.go:295]: same as above

File: cmd/porter/installations.go [hunk 3]
```diff
@@ -206,3 +211,256 @@ func buildInstallationRunsListCommand(p *porter.Porter) *cobra.Command {
...
+Porter uses the docker driver as the default runtime for executing a bundle's invocation image, but an alternate driver may be supplied via '--driver/-d' or the PORTER_RUNTIME_DRIVER environment variable.
+For example, the 'debug' driver may be specified, which simply logs the info given to it and then exits.
+
+The docker driver runs the bundle container using the local Docker host. To use a remote Docker host, set the following environment variables:
+  DOCKER_HOST (required)
+  DOCKER_TLS_VERIFY (optional)
+  DOCKER_CERT_PATH (optional)
+`,
+		Example: `  porter bundle invoke --action ACTION
+  porter bundle invoke --reference ghcr.io/getporter/examples/kubernetes:v0.2.0
```
Comment (VinozzZ) [inline cmd/porter/installations.go:354]: same as above

File: cmd/porter/installations.go [hunk 4]
```diff
@@ -206,3 +211,256 @@ func buildInstallationRunsListCommand(p *porter.Porter) *cobra.Command {
...
+Porter uses the docker driver as the default runtime for executing a bundle's invocation image, but an alternate driver may be supplied via '--driver/-d'' or the PORTER_RUNTIME_DRIVER environment variable.
+For example, the 'debug' driver may be specified, which simply logs the info given to it and then exits.
+
+The docker driver runs the bundle container using the local Docker host. To use a remote Docker host, set the following environment variables:
+  DOCKER_HOST (required)
+  DOCKER_TLS_VERIFY (optional)
+  DOCKER_CERT_PATH (optional)
+`,
+		Example: `  porter bundle uninstall
+  porter bundle uninstall --reference ghcr.io/getporter/examples/kubernetes:v0.2.0
```
Comment (VinozzZ) [inline cmd/porter/installations.go:413]: same as above

File: docs/content/cli/install.md [hunk 5]
```diff
@@ -31,14 +31,14 @@ porter install [INSTALLATION] [flags]
...
 ```
-  porter install
-  porter install MyAppFromReference --reference ghcr.io/getporter/examples/kubernetes:v0.2.0 --namespace dev
-  porter install --reference localhost:5000/ghcr.io/getporter/examples/kubernetes:v0.2.0 --insecure-registry --force
-  porter install MyAppInDev --file myapp/bundle.json
-  porter install --parameter-set azure --param test-mode=true --param header-color=blue
-  porter install --cred azure --cred kubernetes
-  porter install --driver debug
-  porter install --label env=dev --label owner=myuser
+  porter bundle install
```
Comment (VinozzZ) [inline docs/content/cli/install.md:34]: should these be `porter installation install` ?

File: docs/content/cli/invoke.md [hunk 6]
```diff
@@ -29,13 +29,13 @@ porter invoke [INSTALLATION] --action ACTION [flags]
...
 
 ```
-  porter invoke --action ACTION
-  porter invoke --reference ghcr.io/getporter/examples/kubernetes:v0.2.0
-  porter invoke --reference localhost:5000/ghcr.io/getporter/examples/kubernetes:v0.2.0 --insecure-registry --force
-  porter invoke --action ACTION MyAppInDev --file myapp/bundle.json
-  porter invoke --action ACTION  --parameter-set azure --param test-mode=true --param header-color=blue
-  porter invoke --action ACTION --cred azure --cred kubernetes
-  porter invoke --action ACTION --driver debug
+  porter bundle invoke --action ACTION
```
Comment (VinozzZ) [inline docs/content/cli/invoke.md:32]: `porter installation invoke`?

File: docs/content/cli/uninstall.md [hunk 7]
```diff
@@ -29,15 +29,15 @@ porter uninstall [INSTALLATION] [flags]
...
-  porter uninstall
-  porter uninstall --reference ghcr.io/getporter/examples/kubernetes:v0.2.0
-  porter uninstall --reference localhost:5000/ghcr.io/getporter/examples/kubernetes:v0.2.0 --insecure-registry --force
-  porter uninstall MyAppInDev --file myapp/bundle.json
-  porter uninstall --parameter-set azure --param test-mode=true --param header-color=blue
-  porter uninstall --cred azure --cred kubernetes
-  porter uninstall --driver debug
-  porter uninstall --delete
-  porter uninstall --force-delete
+  porter bundle uninstall
```
Comment (VinozzZ) [inline docs/content/cli/uninstall.md:32]: `porter installation uninstall`

File: docs/content/cli/upgrade.md [hunk 8]
```diff
@@ -29,13 +29,13 @@ porter upgrade [INSTALLATION] [flags]
...
 
 ```
-  porter upgrade --version 0.2.0
-  porter upgrade --reference ghcr.io/getporter/examples/kubernetes:v0.2.0
-  porter upgrade --reference localhost:5000/ghcr.io/getporter/examples/kubernetes:v0.2.0 --insecure-registry --force
-  porter upgrade MyAppInDev --file myapp/bundle.json
-  porter upgrade --parameter-set azure --param test-mode=true --param header-color=blue
-  porter upgrade --cred azure --cred kubernetes
-  porter upgrade --driver debug
+  porter bundle upgrade --version 0.2.0
```
Comment (VinozzZ) [inline docs/content/cli/upgrade.md:32]: `porter installation upgrade`

**Review by VinozzZ — CHANGES_REQUESTED** [review]

**Comment by carolynvs** [comment]
@VinozzZ I have updated the examples and regenerated the cli docs.

**Review by VinozzZ — APPROVED** [review]
lgtm, just need to fix the merge conflict

**Comment by carolynvs** [comment]
@VinozzZ I fixed the merge conflict by copying the changes that were originally made to cmd/porter/bundle.go to the newly relocated functions that are now in cmd/porter/installation.go.

File: docs/content/cli/upgrade.md [hunk 9]
```diff
@@ -34,7 +34,7 @@ porter upgrade [INSTALLATION] [flags]
   porter upgrade --reference localhost:5000/ghcr.io/getporter/examples/kubernetes:v0.2.0 --insecure-registry --force
   porter upgrade MyAppInDev --file myapp/bundle.json
   porter upgrade --parameter-set azure --param test-mode=true --param header-color=blue
-  porter upgrade --credential-set azure --credential-set kubernetes
+  porter upgrade --cred azure --cred kubernetes
```
Comment (VinozzZ) [inline docs/content/cli/upgrade.md:37]: I think the doc changes got reverted during the merge

Comment (carolynvs) [reply docs/content/cli/upgrade.md:37]: 😭

Comment (carolynvs) [reply docs/content/cli/upgrade.md]: I pushed up a fix for the bad merge conflict resolution.

**Review by VinozzZ — APPROVED** [review]

**Pull request merged**